src/plugins/add-to-basket-exp-mixin.js

```diff
@@ -1,21 +1,28 @@
-import experimentVersionFragment from '#src/graphql/fragments/experimentVersion.graphql';
+import logReadQueryError from '#src/util/logReadQueryError';
+import { readBoolSetting } from '#src/util/settingsUtils';
+import uiConfigSettingQuery from '#src/graphql/query/uiConfigSetting.graphql';
 
-const NEW_ADD_TO_BASKET_EXP = 'new_add_to_basket';
+const NEW_ADD_TO_BASKET_KEY = 'new_atb_experience_enable';
 
 export default {
+	inject: ['apollo', 'cookieStore'],
```

Comment:
Is `cookieStore` needed?

Reply:
Sometimes added besides apollo, let see if smt changes...